pages/common/sudo.md

```diff
@@ -1,23 +1,23 @@
 # sudo
 
-> Executes a single command as another user.
+> Executes a single command as the superuser or another user.
 
-- List the contents of an unreadable directory:
+- Run a command as the superuser:
 
-`sudo {{ls}} {{/usr/local/scrt}}`
+`sudo {{shutdown now}}`
 
-- Edit a file as the user www:
+- Edit a file as the superuser with your default editor:
 
-`sudo -u {{www}} {{vi}} {{/var/www/index.html}}`
+`sudo -e {{/etc/fstab}}`
 
-- Shut down the machine:
+- Run a command as another user and/or group:
 
-`sudo {{shutdown}} -h +10 {{"Cya soon!"}}`
+`sudo -u {{user}} -g {{group}} {{id}}`
```

Comment:
Maybe we can change `id` to `whoami` to make it more clear ?

Reply:
I actually used `whoami` when I first rewrote the example; however, the trouble with `whoami` is that the output does not change when including `-g` flag in sudo whereas `id`does. Although I do worry that `id` may be mistaken for another variable rather than a command. Is another command besides these two we could use?

Reply:
I didn't even know that executing as another group was a thing! Could we use an `echo` command to output the contents of an environment variable?

Reply:
Yea, even I thought id was another flag or something. If there is no better option, then I am fine with it.

Reply:
I've thought about this some more and I can't come up with another command that is as succinct as `id` and shows the effects of `sudo` as well as `id` does. One possibility to make it clearer that it is a command rather than another flag to sudo is to use `id -a` instead. The `-a` flag is ignored in most versions, but it's there for compatibility reasons. What do you guys think?

Reply:
Great, let's do that.